common/changes/overflow-set-overflowitems_2017-05-11-04-52.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
## [🕵 fluentai-react Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=1760&commitId=78a278a443078c11fb9a153c1c93f81820094ba4&env=prod&clientType=FLUENTUI)


  <details><summary><b>Components-AiGeneratedDisclaimer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-AiGeneratedDisclaimer.Default.chromium.png|0| Added|

</details>


  <details><summary><b>Components-AiGeneratedDisclaimer-Tests-Visual Regression</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-AiGeneratedDisclaimer-Tests-Visual Regression.AiGeneratedDisclaimerDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Attachments-AttachmentTag</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Attachments-AttachmentTag.Default.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Attachments-AttachmentTag-Tests-Visual Regression</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Attachments-AttachmentTag-Tests-Visual Regression.Attachment Tag Default Focus.chromium.png|0| Added|
|Components-Attachments-AttachmentTag-Tests-Visual Regression.Attachment Tag Default Hover.chromium.png|0| Added|
|Components-Attachments-AttachmentTag-Tests-Visual Regression.AttachmentTagDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Attachments-AttachmentTagItem</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Attachments-AttachmentTagItem.Default.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Attachments-AttachmentTagItem-Tests-Visual Regression</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Attachments-AttachmentTagItem-Tests-Visual Regression.AttachmentTagItemDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Attachments-AttachmentTagList</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Attachments-AttachmentTagList.AttachmentTagListOverflow.chromium.png|0| Added|
|Components-Attachments-AttachmentTagList.Default.chromium.png|0| Added|
|Components-Attachments-AttachmentTagList.Dismissal.chromium.png|0| Added|
|Components-Attachments-AttachmentTagList.Wrapping.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Attachments-AttachmentTagList-Tests-Visual Regression</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Attachments-AttachmentTagList-Tests-Visual Regression.AttachmentTagListDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-ChatInput</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-ChatInput.Attachments.chromium.png|0| Added|
|Components-ChatInput.AutoExpand.chromium.png|0| Added|
|Components-ChatInput.Autocomplete.chromium.png|0| Added|
|Components-ChatInput.DebugInfo.chromium.png|0| Added|
|Components-ChatInput.Default.chromium.png|0| Added|
|Components-ChatInput.Disabled.chromium.png|0| Added|
|Components-ChatInput.Entities.chromium.png|0| Added|
|Components-ChatInput.ErrorMessage.chromium.png|0| Added|
|Components-ChatInput.ImperativeControl.chromium.png|0| Added|
|Components-ChatInput.PasteEntity.chromium.png|0| Added|
|Components-ChatInput.PasteUnfurling.chromium.png|0| Added|
|Components-ChatInput.SlashPlaceholder.chromium.png|0| Added|
|Components-ChatInput.Tokens.chromium.png|0| Added|

</details>


  <details><summary><b>Components-ChatInput-ChatInputPlugins-ChatInputEntity</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-ChatInput-ChatInputPlugins-ChatInputEntity.Anchor.chromium.png|0| Added|
|Components-ChatInput-ChatInputPlugins-ChatInputEntity.Default.chromium.png|0| Added|
|Components-ChatInput-ChatInputPlugins-ChatInputEntity.Sensitivity.chromium.png|0| Added|

</details>


  <details><summary><b>Components-ChatInput-ChatInputPlugins-ChatInputToken</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-ChatInput-ChatInputPlugins-ChatInputToken.Default.chromium.png|0| Added|

</details>


  <details><summary><b>Components-ChatInput-ChatInputPlugins-ChatInputToken-Tests-Visual Regression</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-ChatInput-ChatInputPlugins-ChatInputToken-Tests-Visual Regression.ChatInputTokenDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-ChatInput-ChatInputPlugins-GhostText</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-ChatInput-ChatInputPlugins-GhostText.Default.chromium.png|0| Added|

</details>


  <details><summary><b>Components-ChatInput-Tests-Visual Regression</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-ChatInput-Tests-Visual Regression.Chat Input Auto Expand Focus.chromium.png|0| Added|
|Components-ChatInput-Tests-Visual Regression.Chat Input Auto Expand Hover.chromium.png|0| Added|
|Components-ChatInput-Tests-Visual Regression.Chat Input Entities Sensitivity.chromium.png|0| Added|

</details>


  <details><summary><b>Components-ChatInput-Writing Plugins</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-ChatInput-Writing Plugins.Docs.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Citation</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Citation.Default.chromium.png|0| Added|
|Components-Citations and References-Citation.Hook.chromium.png|0| Added|
|Components-Citations and References-Citation.Multiple.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Citation-Tests-End to End</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Citation-Tests-End to End.Citation Hook Click.chromium.png|0| Added|
|Components-Citations and References-Citation-Tests-End to End.Citation Hook Enter.chromium.png|0| Added|
|Components-Citations and References-Citation-Tests-End to End.Citation Hook Hover.chromium.png|0| Added|
|Components-Citations and References-Citation-Tests-End to End.Citation Hook Popover Close.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Citation-Tests-Visual Regression</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Citation-Tests-Visual Regression.Citation Default Focus.chromium.png|0| Added|
|Components-Citations and References-Citation-Tests-Visual Regression.Citation Default Hover.chromium.png|0| Added|
|Components-Citations and References-Citation-Tests-Visual Regression.CitationDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Reference-Deprecated-Reference</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Reference-Deprecated-Reference.Default.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-Reference.Hook.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-Reference.WithSensitivityLabel.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Reference-Deprecated-Reference-Tests-End to End</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Reference-Deprecated-Reference-Tests-End to End.Citation Hook Enter.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-Reference-Tests-End to End.Citation Hook Popover Close.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-Reference-Tests-End to End.Reference Hook Click.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-Reference-Tests-End to End.Reference Hook Hover.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Reference-Deprecated-Reference-Tests-Visual Regression</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Reference-Deprecated-Reference-Tests-Visual Regression.Reference Default Expanded.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-Reference-Tests-Visual Regression.Reference Default Focus.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-Reference-Tests-Visual Regression.Reference Default Hover.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-Reference-Tests-Visual Regression.ReferenceDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Reference-Deprecated-ReferenceGroup-ReferenceGroupToggle</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Reference-Deprecated-ReferenceGroup-ReferenceGroupToggle.ControlledState.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-ReferenceGroup-ReferenceGroupToggle.Default.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-ReferenceGroup-ReferenceGroupToggle.Hook.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Reference-Deprecated-ReferenceGroup-ReferenceGroupToggle-Tests-Visual Regression</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Reference-Deprecated-ReferenceGroup-ReferenceGroupToggle-Tests-Visual Regression.Reference Group Toggle Open.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-ReferenceGroup-ReferenceGroupToggle-Tests-Visual Regression.ReferenceGroupToggleDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Reference-Deprecated-ReferenceGroup-ReferenceList</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Reference-Deprecated-ReferenceGroup-ReferenceList.Default.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Reference-Deprecated-ReferenceGroup-ReferenceList-Tests-Visual Regression</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Reference-Deprecated-ReferenceGroup-ReferenceList-Tests-Visual Regression.Reference List Default Expanded.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-ReferenceGroup-ReferenceList-Tests-Visual Regression.ReferenceListDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Reference-Deprecated-ReferenceGroup-useReferenceGroup</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Reference-Deprecated-ReferenceGroup-useReferenceGroup.Default.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Reference-Deprecated-ReferenceGroup-useReferenceGroup-Tests-Visual Regression</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Reference-Deprecated-ReferenceGroup-useReferenceGroup-Tests-Visual Regression.Use Reference Group Default Focus.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-ReferenceGroup-useReferenceGroup-Tests-Visual Regression.Use Reference Group Expanded.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-ReferenceGroup-useReferenceGroup-Tests-Visual Regression.Use Reference Group Keyboard Expanded.chromium.png|0| Added|
|Components-Citations and References-Reference-Deprecated-ReferenceGroup-useReferenceGroup-Tests-Visual Regression.UseReferenceGroupDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Reference-v2-Reference</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Reference-v2-Reference.Default.chromium.png|0| Added|
|Components-Citations and References-Reference-v2-Reference.Hook.chromium.png|0| Added|
|Components-Citations and References-Reference-v2-Reference.WithPreview.chromium.png|0| Added|
|Components-Citations and References-Reference-v2-Reference.WithPreviewLink.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Reference-v2-Reference-Tests-Visual Regression</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Reference-v2-Reference-Tests-Visual Regression.Reference Default Focus.chromium.png|0| Added|
|Components-Citations and References-Reference-v2-Reference-Tests-Visual Regression.Reference Default Hover.chromium.png|0| Added|
|Components-Citations and References-Reference-v2-Reference-Tests-Visual Regression.Reference Default Rest.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Reference-v2-ReferenceList</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Reference-v2-ReferenceList.Default.chromium.png|0| Added|
|Components-Citations and References-Reference-v2-ReferenceList.WithOverflow.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Citations and References-Reference-v2-ReferenceOverflowButton</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Citations and References-Reference-v2-ReferenceOverflowButton.Default.chromium.png|0| Added|

</details>


  <details><summary><b>Components-CopilotChat</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-CopilotChat.CustomAnnouncement.chromium.png|0| Added|
|Components-CopilotChat.Default.chromium.png|0| Added|

</details>


  <details><summary><b>Components-CopilotChat-Tests-Visual Regression</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-CopilotChat-Tests-Visual Regression.Copilot Chat Default Focus.chromium.png|0| Added|
|Components-CopilotChat-Tests-Visual Regression.CopilotChatDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-CopilotMessageFooter</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-CopilotMessageFooter.Default.chromium.png|0| Added|
|Components-CopilotMessageFooter.WithContent.chromium.png|0| Added|

</details>


  <details><summary><b>Components-CopilotMessageFooter-Tests-Visual Regression</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-CopilotMessageFooter-Tests-Visual Regression.Copilot Message Footer Default Focus.chromium.png|0| Added|
|Components-CopilotMessageFooter-Tests-Visual Regression.Copilot Message Footer Default Hover.chromium.png|0| Added|
|Components-CopilotMessageFooter-Tests-Visual Regression.Copilot Message Footer References Expanded.chromium.png|0| Added|
|Components-CopilotMessageFooter-Tests-Visual Regression.Copilot Message Footer Sensitivity Expanded.chromium.png|0| Added|

</details>


  <details><summary><b>Components-CopilotProvider</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-CopilotProvider.Default.chromium.png|0| Added|

</details>


  <details><summary><b>Components-FeedbackButtons</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-FeedbackButtons.CustomTooltips.chromium.png|0| Added|
|Components-FeedbackButtons.Default.chromium.png|0| Added|
|Components-FeedbackButtons.DefaultWithDialog.chromium.png|0| Added|
|Components-FeedbackButtons.Disabled.chromium.png|0| Added|

</details>


  <details><summary><b>Components-FeedbackButtons-Tests-Visual Regression</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-FeedbackButtons-Tests-Visual Regression.Feedback Buttons Default Focus.chromium.png|0| Added|
|Components-FeedbackButtons-Tests-Visual Regression.Feedback Buttons Default Hover.chromium.png|0| Added|
|Components-FeedbackButtons-Tests-Visual Regression.Feedback Buttons Default Negative.chromium.png|0| Added|
|Components-FeedbackButtons-Tests-Visual Regression.Feedback Buttons Default Positive.chromium.png|0| Added|
|Components-FeedbackButtons-Tests-Visual Regression.FeedbackButtonsDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-FirstRunExperience</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-FirstRunExperience.Controlled.chromium.png|0| Added|
|Components-FirstRunExperience.Default.chromium.png|0| Added|
|Components-FirstRunExperience.NoSurface.chromium.png|0| Added|

</details>


  <details><summary><b>Components-FirstRunExperience-Tests-Visual Regression</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-FirstRunExperience-Tests-Visual Regression.First Run Experience Default First Slide.chromium.png|0| Added|
|Components-FirstRunExperience-Tests-Visual Regression.First Run Experience Default Last Slide.chromium.png|0| Added|

</details>


  <details><summary><b>Components-LatencyLoader</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-LatencyLoader.Default.chromium.png|0| Added|
|Components-LatencyLoader.DeterminateLatencyLoader.chromium.png|0| Added|
|Components-LatencyLoader.LatencyLoaderWithChildren.chromium.png|0| Added|
|Components-LatencyLoader.LatencyLoaderWithUpdates.chromium.png|0| Added|

</details>


  <details><summary><b>Components-LatencyLoader-Tests-Visual Regression</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-LatencyLoader-Tests-Visual Regression.Latency Loader Default Determinate.chromium.png|0| Added|

</details>


  <details><summary><b>Components-OutputCard</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-OutputCard.Default.chromium.png|0| Added|
|Components-OutputCard.ManualFlair.chromium.png|0| Added|

</details>


  <details><summary><b>Components-OutputCard-Tests-Visual Regression</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-OutputCard-Tests-Visual Regression.Output Card Default Loading.chromium.png|0| Added|
|Components-OutputCard-Tests-Visual Regression.OutputCardDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Preview</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Preview.Navigation.chromium.png|0| Added|
|Components-Preview.Thumbnail.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Preview-Tests-Visual Regression</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Preview-Tests-Visual Regression.Preview Open.chromium.png|0| Added|

</details>


  <details><summary><b>Components-PromptStarter</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-PromptStarter.Default.chromium.png|0| Added|
|Components-PromptStarter.ZeroPromptExample.chromium.png|0| Added|

</details>


  <details><summary><b>Components-PromptStarter-Tests-Visual Regression</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-PromptStarter-Tests-Visual Regression.Prompt Starter Default Focus.chromium.png|0| Added|
|Components-PromptStarter-Tests-Visual Regression.Prompt Starter Default Hover.chromium.png|0| Added|
|Components-PromptStarter-Tests-Visual Regression.PromptStarterDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-ResponseCount</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-ResponseCount.Default.chromium.png|0| Added|
|Components-ResponseCount.Status.chromium.png|0| Added|

</details>


  <details><summary><b>Components-ResponseCount-Tests-Visual Regression</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-ResponseCount-Tests-Visual Regression.Response Count Danger.chromium.png|0| Added|
|Components-ResponseCount-Tests-Visual Regression.Response Count Success.chromium.png|0| Added|
|Components-ResponseCount-Tests-Visual Regression.Response Count Warning.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Sensitivity-SensitivityLabel</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Sensitivity-SensitivityLabel.Default.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Sensitivity-SensitivityLabel-Tests-Visual Regression</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Sensitivity-SensitivityLabel-Tests-Visual Regression.Sensitivity Label Default Focus.chromium.png|0| Added|
|Components-Sensitivity-SensitivityLabel-Tests-Visual Regression.Sensitivity Label Default Hover.chromium.png|0| Added|
|Components-Sensitivity-SensitivityLabel-Tests-Visual Regression.Sensitivity Label Default Open.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Sensitivity-SensitivityTooltip</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Sensitivity-SensitivityTooltip.Backplate.chromium.png|0| Added|
|Components-Sensitivity-SensitivityTooltip.Default.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Sensitivity-SensitivityTooltip-Tests-Visual Regression</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Sensitivity-SensitivityTooltip-Tests-Visual Regression.Sensitivity Tooltip Default Focus.chromium.png|0| Added|
|Components-Sensitivity-SensitivityTooltip-Tests-Visual Regression.Sensitivity Tooltip Default Hover.chromium.png|0| Added|
|Components-Sensitivity-SensitivityTooltip-Tests-Visual Regression.Sensitivity Tooltip Default Open.chromium.png|0| Added|
|Components-Sensitivity-SensitivityTooltip-Tests-Visual Regression.Sensitivity Tooltip With Backplate Focus.chromium.png|0| Added|
|Components-Sensitivity-SensitivityTooltip-Tests-Visual Regression.Sensitivity Tooltip With Backplate Hover.chromium.png|0| Added|
|Components-Sensitivity-SensitivityTooltip-Tests-Visual Regression.Sensitivity Tooltip With Backplate Open.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Snippet</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Snippet.Default.chromium.png|0| Added|
|Components-Snippet.SnippetSyntax.chromium.png|0| Added|
|Components-Snippet.SnippetSyntaxWithDropdowns.chromium.png|0| Added|
|Components-Snippet.WithDropdowns.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Snippet-Tests-Visual Regression</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Snippet-Tests-Visual Regression.Snippet Default Focus.chromium.png|0| Added|
|Components-Snippet-Tests-Visual Regression.Snippet Default Hover.chromium.png|0| Added|
|Components-Snippet-Tests-Visual Regression.SnippetDefault.chromium.png|0| Added|
|Components-Snippet-Tests-Visual Regression.SnippetWithFooter.chromium.png|0| Added|
|Components-Snippet-Tests-Visual Regression.SnippetWithHeader.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Suggestions</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Suggestions.Default.chromium.png|0| Added|
|Components-Suggestions.SuggestionList.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Suggestions-Tests-Visual Regression</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Suggestions-Tests-Visual Regression.Suggestion List Default Focus.chromium.png|0| Added|
|Components-Suggestions-Tests-Visual Regression.Suggestion List Default Hover.chromium.png|0| Added|
|Components-Suggestions-Tests-Visual Regression.SuggestionListDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-SystemMessage</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-SystemMessage.Default.chromium.png|0| Added|
|Components-SystemMessage.InChat.chromium.png|0| Added|
|Components-SystemMessage.NoIcon.chromium.png|0| Added|
|Components-SystemMessage.WithPluginName.chromium.png|0| Added|
|Components-SystemMessage.Wrapping.chromium.png|0| Added|

</details>


  <details><summary><b>Components-SystemMessage-Tests-Visual Regression</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-SystemMessage-Tests-Visual Regression.System Message Default Focus.chromium.png|0| Added|
|Components-SystemMessage-Tests-Visual Regression.System Message No Icon.chromium.png|0| Added|
|Components-SystemMessage-Tests-Visual Regression.System Message Wrapping.chromium.png|0| Added|
|Components-SystemMessage-Tests-Visual Regression.SystemMessageDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Textarea</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Textarea.AutoAppearance.chromium.png|0| Added|
|Components-Textarea.Default.chromium.png|0| Added|
|Components-Textarea.Disabled.chromium.png|0| Added|
|Components-Textarea.SendDisabled.chromium.png|0| Added|
|Components-Textarea.Single.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Textarea-Tests-Visual Regression</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Textarea-Tests-Visual Regression.Textarea Default Focus.chromium.png|0| Added|
|Components-Textarea-Tests-Visual Regression.Textarea Default Hover.chromium.png|0| Added|
|Components-Textarea-Tests-Visual Regression.Textarea Default New Line.chromium.png|0| Added|
|Components-Textarea-Tests-Visual Regression.Textarea Send Disabled.chromium.png|0| Added|
|Components-Textarea-Tests-Visual Regression.Textarea Single.chromium.png|0| Added|
|Components-Textarea-Tests-Visual Regression.TextareaDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Timestamp</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Timestamp.Default.chromium.png|0| Added|
|Components-Timestamp.InChat.chromium.png|0| Added|

</details>


  <details><summary><b>Components-Timestamp-Tests-Visual Regression</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Components-Timestamp-Tests-Visual Regression.TimestampDefault.chromium.png|0| Added|

</details>


  <details><summary><b>Experiences-Chat</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Experiences-Chat.CopilotChatExperience.chromium.png|0| Added|

</details>


  <details><summary><b>Get started-Accessibility</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Get started-Accessibility.Docs.chromium.png|0| Added|

</details>


  <details><summary><b>Get started-Design</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Get started-Design.Docs.chromium.png|0| Added|

</details>


  <details><summary><b>Get started-Develop</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Get started-Develop.Docs.chromium.png|0| Added|

</details>


  <details><summary><b>Get started-New Package Scope</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Get started-New Package Scope.Docs.chromium.png|0| Added|

</details>


  <details><summary><b>In Development-Attachments-Attachment</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|In Development-Attachments-Attachment.Default.chromium.png|0| Added|
|In Development-Attachments-Attachment.DismissOnly.chromium.png|0| Added|
|In Development-Attachments-Attachment.ImageOnly.chromium.png|0| Added|
|In Development-Attachments-Attachment.withProgressBar.chromium.png|0| Added|

</details>


  <details><summary><b>In Development-Attachments-Attachment-Tests-Visual Regression</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|In Development-Attachments-Attachment-Tests-Visual Regression.Attachment Default Focus Dismiss.chromium.png|0| Added|
|In Development-Attachments-Attachment-Tests-Visual Regression.Attachment Default Focus Primary.chromium.png|0| Added|
|In Development-Attachments-Attachment-Tests-Visual Regression.Attachment Default Hover.chromium.png|0| Added|
|In Development-Attachments-Attachment-Tests-Visual Regression.Attachment Dismiss Only Hover.chromium.png|0| Added|
|In Development-Attachments-Attachment-Tests-Visual Regression.Attachment Dismiss Only.chromium.png|0| Added|
|In Development-Attachments-Attachment-Tests-Visual Regression.AttachmentDefault.chromium.png|0| Added|

</details>


  <details><summary><b>In Development-Attachments-AttachmentList</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|In Development-Attachments-AttachmentList.Default.chromium.png|0| Added|
|In Development-Attachments-AttachmentList.Dismissal.chromium.png|0| Added|
|In Development-Attachments-AttachmentList.MultipleProgressBars.chromium.png|0| Added|
|In Development-Attachments-AttachmentList.WithOverflow.chromium.png|0| Added|
|In Development-Attachments-AttachmentList.WithOverflowCustomButton.chromium.png|0| Added|

</details>


  <details><summary><b>In Development-CopilotChat-v2-CopilotChat</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|In Development-CopilotChat-v2-CopilotChat.CustomAnnouncement.chromium.png|0| Added|
|In Development-CopilotChat-v2-CopilotChat.Default.chromium.png|0| Added|

</details>


  <details><summary><b>In Development-CopilotChat-v2-CopilotChat-Tests-Visual Regression</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|In Development-CopilotChat-v2-CopilotChat-Tests-Visual Regression.Copilot Chat Default Focus.chromium.png|0| Added|
|In Development-CopilotChat-v2-CopilotChat-Tests-Visual Regression.CopilotChatDefault.chromium.png|0| Added|

</details>


  <details><summary><b>In Development-EditorInput</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|In Development-EditorInput.Default.chromium.png|0| Added|

</details>


  <details><summary><b>In Development-Keep</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|In Development-Keep.Default.chromium.png|0| Added|

</details>


  <details><summary><b>In Development-PromptInput</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|In Development-PromptInput.PromptInputAppearance.chromium.png|0| Added|
|In Development-PromptInput.PromptInputAttachments.chromium.png|0| Added|
|In Development-PromptInput.PromptInputDefault.chromium.png|0| Added|
|In Development-PromptInput.PromptInputDisabled.chromium.png|0| Added|
|In Development-PromptInput.PromptInputError.chromium.png|0| Added|

</details>


  <details><summary><b>In Development-SensitivityIcon</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|In Development-SensitivityIcon.Default.chromium.png|0| Added|
|In Development-SensitivityIcon.Filled.chromium.png|0| Added|
|In Development-SensitivityIcon.FilledWithLock.chromium.png|0| Added|
|In Development-SensitivityIcon.WithLock.chromium.png|0| Added|

</details>


  <details><summary><b>Utilities-Announce</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Utilities-Announce.Default.chromium.png|0| Added|

</details>


  <details><summary><b>Utilities-Flair</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Utilities-Flair.ColorFormat.chromium.png|0| Added|
|Utilities-Flair.Default.chromium.png|0| Added|
|Utilities-Flair.Override.chromium.png|0| Added|
|Utilities-Flair.Resize.chromium.png|0| Added|

</details>


  <details><summary><b>Welcome</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Welcome.Docs.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ai_VRT_Pipeline_react"/>